scala-thoughts.md

```diff
@@ -0,0 +1,44 @@
+Scala makes it easy to write functional style code. I like the way this is
+encouraged by the language. For example, the ability to write a function that
+does not include an explicit return statement subtly encourages the programmer
+to think in a functional mindset, by thinking of a function body as an
```

Comment:
That's a really interesting point. I was wondering why this was a feature of the language, but this makes a lot of sense.